hotrod/src/main/java/io/hyperfoil/hotrod/steps/HotRodResponseStep.java

```diff
@@ -0,0 +1,24 @@
+package io.hyperfoil.hotrod.steps;
+
+import io.hyperfoil.api.session.Session;
+import io.hyperfoil.api.statistics.Statistics;
+import io.hyperfoil.core.steps.StatisticsStep;
+import io.hyperfoil.hotrod.resource.HotRodResource;
+import io.hyperfoil.hotrod.resource.HotRodResourceKey;
+
+public class HotRodResponseStep extends StatisticsStep {
```

Comment:
I guess this class does not need to be a `StatisticsStep` anymore as it does not record statistics/define SLA.

Reply:
Done